packages/a11y/README.md

```diff
@@ -16,20 +16,16 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
-### setup
-
-[src/index.js#L16-L26](src/index.js#L16-L26)
+<a name="setup" href="#setup">#</a> **setup** [\<>](src/index.js#L16-L26)
```

Comment:
Maybe not to be addressed here, but the relative links for code are dead links in NPM:

https://www.npmjs.com/package/@wordpress/a11y

It appears NPM _tries_ to append it to the GitHub repository, and it may be that once they properly support the new [`repository.directory`](https://github.com/npm/cli/releases/tag/v6.8.0) on their website it becomes fixed.

Reply:
Curious, was there some prior art for the `<>` ? Is it meant to be an abbreviated message of "this is the code!" ? Once I understood it, it seemed fine enough. But I'd be lying if I said my initial reaction wasn't to consider it as being some erroneous output.

Reply:
Yeah, there is an issue for the links. I plan to address it next.

The `<>` idea I took it from D3 API docs such as: https://github.com/d3/d3-brush/blob/master/README.md#api-reference